saq/email.py

```diff
@@ -18,8 +19,13 @@
 def normalize_email_address(email_address):
     """Returns a normalized version of email address.  Returns None if the address cannot be parsed."""
     name, address = parseaddr(email_address)
-    if address is None:
-        return None
+    if address is '':
```

Comment:
This should be address == '', you generally don't want to use the "is" operator for string comparisons.

Reply:
ah yes, good catch. I apparently wasn't thinking. Will fix